themes-default/slim/views/vue-components/quality-chooser.mako

```diff
@@ -16,7 +16,7 @@
                 </div>
                 <div style="text-align: left; float: left;">
                     <h5>Preferred</h5>
-                    <select v-model="preferredQualities" name="preferred_qualities" multiple="multiple" :size="preferredQualityList.length" class="form-control form-control-inline input-sm">
+                    <select v-model="preferredQualities" name="preferred_qualities" multiple="multiple" :size="preferredQualityList.length" class="form-control form-control-inline input-sm" :disabled="!allowedExplanation.length">
```

Comment:
Since we're not actually checking for the explanation to be empty,
it would be cleaner to do the following check:
`:disabled="allowedQualities.length === 0"`
This also fits better with our coding style (IMO), but `!allowedQualities.length` will work as well.